### PR TITLE
Update MMU RO Language

### PR DESCRIPTION
**Pull request by Hauzman**
Co-Authored-By: Alex Voinea <voinea.dragos.alexandru@gmail.com>

I have updated the Ro Language to MMU2 but didn't check on the printer.
I will have again a double check to see if all is OK

### CODE REVIEW

File: lang/po/Firmware_ro.po [hunk 1]
```diff
@@ -2119,26 +2134,28 @@ msgstr "Temperaturi"
...
 msgstr ""
+"Versiune FW MMU este incompatibil cu cea a imprimantei."
+"Asigurati-va ca FW MMU este actualizat."
 
 #. MSG_DESC_SELECTOR_CANNOT_HOME c=20 r=8
 #: ../../Firmware/mmu2/errors_list.h:202 ../../Firmware/mmu2/errors_list.h:242
 msgid ""
 "The Selector cannot home properly. Check for anything blocking its movement."
-msgstr ""
+msgstr "Selector-ul nu poate reveni" corect. Verificati daca exista ceva care blocheza miscarea.
```
Comment (3d-gussner) [inline lang/po/Firmware_ro.po]: Double quotes wrong. Here the fix.
```
#. MSG_DESC_SELECTOR_CANNOT_HOME c=20 r=8
#: ../../Firmware/mmu2/errors_list.h:202 ../../Firmware/mmu2/errors_list.h:242
msgid ""
"The Selector cannot home properly. Check for anything blocking its movement."
msgstr ""
"Selector-ul nu poate reveni corect. Verificati daca exista ceva care blocheza"
" miscarea."
```

**Review by 3d-gussner — CHANGES_REQUESTED** [review]
@Hauzman @leptun Please address these messages with [W]arnings and [E]rrors.

No empty translations anymore which means if the translation is the same as origin copy/paste it. 

Let me know if you really can't shorten the messages to two screens c=20 r=8. Is it allowed in Romanian to split long words?
This helped me in German where you can have really long words like `Rindfleischetikettierungsüberwachungsaufgabenübertragungsgesetz`
Which can be split into
```
01234567890123456789
Rindfleisch-
etikettierungs-
überwachungs-
aufgaben-
übertragungsgesetz
``` 
```
Prusa-Firmware/MK3/lang$ ./lang-check.py --map firmware_MK3S.map po/Firmware_ro.po --no-suggest
[W]: Empty translation for "Auto" on line 83
[W]: Empty translation for "Auto home" on line 88
[W]: Empty translation for "Avoiding grind" on line 113
[W]: Empty translation for "Cont." on line 356
[W]: Empty translation for "Cutter" on line 407
[W]: Empty translation for "E-correct:" on line 465
[W]: Empty translation for "Endstop" on line 512
[W]: Empty translation for "Engaging idler" on line 527
[W]: Empty translation for "Extruder" on line 534
[W]: Incorrect trailing whitespace for translation on line 564:
 source text:
    ₀₁₂₃₄₅₆₇₈₉₀₁₂
   |F. runout    |
 translated text:
    ₀₁₂₃₄₅₆₇₈₉₀₁₂
   |F. epuizat·  |

[W]: Incorrect trailing whitespace for translation on line 580:
 source text:
      ₀₁₂₃₄₅₆₇₈₉₀₁₂₃₄₅₆₇₈₉
   1 |FINDA didn't switch |
   2 |off while unloading |
   3 |filament. Try       |
   4 |unloading manually. |
   5 |Ensure filament can |
   6 |move and FINDA      |
   7 |works.              |
 translated text:
      ₀₁₂₃₄₅₆₇₈₉₀₁₂₃₄₅₆₇₈₉
   1 |Asigurativa ca      |
   2 |filamentul se misca |
   3 |si FINDA            |
   4 |functioneaza        |

[W]: Empty translation for "FW RUNTIME ERROR" on line 621
[E]: Text is longer than definition on line 672: cols=20 rows=1 (rows diff=1)
 source text:
    ₀₁₂₃₄₅₆₇₈₉₀₁₂₃₄₅₆₇₈₉
   |Feeding to extruder |
 translated text:
    ₀₁₂₃₄₅₆₇₈₉₀₁₂₃₄₅₆₇₈₉
   |Incarcare la extrude|r

[W]: Empty translation for "Filament" on line 698
[E]: Text is longer than definition on line 740: cols=20 rows=8 (rows diff=1)
 source text:
      ₀₁₂₃₄₅₆₇₈₉₀₁₂₃₄₅₆₇₈₉
   1 |Filament sensor     |
   2 |triggered too early |
   3 |while loading to    |
   4 |extruder. Check     |
   5 |there isn't anything|
   6 |stuck in PTFE tube. |
   7 |Check that sensor   |
   8 |reads properly.     |
 translated text:
      ₀₁₂₃₄₅₆₇₈₉₀₁₂₃₄₅₆₇₈₉
   1 |FSensoz sa declansat|
   2 |prea devreme in     |
   3 |timpul incarcarii in|
   4 |expruder. Verifica  |
   5 |dacanu este nimic   |
   6 |blocat in PTFE.     |
   7 |Verifica daca       |
   8 |senzorul            |
   9 |functioneaza corect.|

[W]: Empty translation for "Flow" on line 778
[W]: Empty translation for "Homing" on line 904
[W]: Empty translation for "IDLER CANNOT HOME" on line 920
[E]: Text is longer than definition on line 1059: cols=18 rows=1 (rows diff=1)
 source text:
    ₀₁₂₃₄₅₆₇₈₉₀₁₂₃₄₅₆₇
   |Load to extruder  |
 translated text:
    ₀₁₂₃₄₅₆₇₈₉₀₁₂₃₄₅₆₇
   |Incarca la extrude|r

[W]: Empty translation on line 1098
      ₀₁₂₃₄₅₆₇₈₉₀₁₂₃₄₅₆₇₈₉
   1 |MMU Firmware        |
   2 |internal error,     |
   3 |please reset the    |
   4 |MMU.                |

[W]: Empty translation for "Mesh" on line 1180
[W]: Empty translation for "Model" on line 1207
[W]: Empty translation for "More⏬" on line 1228
[W]: Empty translation for "Motor" on line 1234
[W]: Empty translation for "N/A" on line 1266
[W]: Empty translation for "Normal" on line 1306
[W]: Empty translation for "OK" on line 1353
[W]: Empty translation for "Off" on line 1359
[W]: Empty translation for "On" on line 1378
[W]: Empty translation for "PAUSED THERMAL ERROR" on line 1396
[E]: Text is longer than definition on line 1443: cols=20 rows=1 (rows diff=1)
 source text:
    ₀₁₂₃₄₅₆₇₈₉₀₁₂₃₄₅₆₇₈₉
   |PULLEY CANNOT MOVE  |
 translated text:
    ₀₁₂₃₄₅₆₇₈₉₀₁₂₃₄₅₆₇₈₉
   |PULLEY NU SE POATE M|ISCA

[W]: Empty translation for "Preparing blade" on line 1628
[W]: Empty translation for "QUEUE FULL" on line 1709
[E]: Text is longer than definition on line 1740: cols=20 rows=8 (rows diff=1)
 source text:
      ₀₁₂₃₄₅₆₇₈₉₀₁₂₃₄₅₆₇₈₉
   1 |Requested filament  |
   2 |tool is not         |
   3 |available on this   |
   4 |hardware. Check the |
   5 |G-code for tool     |
   6 |index out of range  |
   7 |(T0-T4).            |
 translated text:
      ₀₁₂₃₄₅₆₇₈₉₀₁₂₃₄₅₆₇₈₉
   1 |Instrumentul de     |
   2 |filament solicitat  |
   3 |nu este disponibil  |
   4 |pe acest hardware.  |
   5 |Verificati G-codul  |
   6 |pentru indicele     |
   7 |instrumentului in   |
   8 |afara intervalului  |
   9 |(T0-T4).            |

[W]: Empty translation for "Reset MMU" on line 1755
[W]: Empty translation for "Retry" on line 1782
[W]: Empty translation for "SELECTOR CANNOT HOME" on line 1819
[W]: Empty translation for "Stop" on line 2051
[W]: Empty translation for "Strict" on line 2062
[W]: Empty translation for "THERMAL ANOMALY" on line 2079
[W]: Empty translation for "TMC DRIVER SHORTED" on line 2101
[E]: Text is longer than definition on line 2107: cols=20 rows=1 (rows diff=1)
 source text:
    ₀₁₂₃₄₅₆₇₈₉₀₁₂₃₄₅₆₇₈₉
   |TMC OVERHEAT ERROR  |
 translated text:
    ₀₁₂₃₄₅₆₇₈₉₀₁₂₃₄₅₆₇₈₉
   |ERR TMC supraincalzi|t 

[W]: Incorrect trailing whitespace for translation on line 2107:
 source text:
    ₀₁₂₃₄₅₆₇₈₉₀₁₂₃₄₅₆₇₈₉
   |TMC OVERHEAT ERROR  |
 translated text:
    ₀₁₂₃₄₅₆₇₈₉₀₁₂₃₄₅₆₇₈₉
   |ERR TMC supraincalzi|t·

[W]: Empty translation for "Temp. model autotune" on line 2119
[W]: Empty translation for "Timeout" on line 2186
[W]: Empty translation for "Total" on line 2191
[W]: Empty translation on line 2223
      ₀₁₂₃₄₅₆₇₈₉₀₁₂₃₄₅₆₇₈₉
   1 |Unexpected FINDA    |
   2 |reading. Ensure no  |
   3 |filament is under   |
   4 |FINDA and the       |
   5 |selector is free.   |
   6 |Check FINDA         |
   7 |connection.         |

[W]: Empty translation for "Unload" on line 2230
[E]: Text is longer than definition on line 2235: cols=16 rows=1 (rows diff=1)
 source text:
    ₀₁₂₃₄₅₆₇₈₉₀₁₂₃₄₅
   |Unload filament |
 translated text:
    ₀₁₂₃₄₅₆₇₈₉₀₁₂₃₄₅
   |Descarca filamen|t

[E]: Text is longer than definition on line 2270: cols=20 rows=1 (rows diff=1)
 source text:
    ₀₁₂₃₄₅₆₇₈₉₀₁₂₃₄₅₆₇₈₉
   |WARNING TMC TOO HOT |
 translated text:
    ₀₁₂₃₄₅₆₇₈₉₀₁₂₃₄₅₆₇₈₉
   |ATENTIE TMC suprainc|alzit

[W]: Empty translation for "Wizard" on line 2338
[E]: Text is longer than definition on line 2442: cols=20 rows=8 (rows diff=1)
 source text:
      ₀₁₂₃₄₅₆₇₈₉₀₁₂₃₄₅₆₇₈₉
   1 |You are using       |
   2 |firmware alpha      |
   3 |version. This is    |
   4 |development version.|
   5 |Using this version  |
   6 |is not recommended  |
   7 |and may cause       |
   8 |printer damage.     |
 translated text:
      ₀₁₂₃₄₅₆₇₈₉₀₁₂₃₄₅₆₇₈₉
   1 |Utilizati versiunea |
   2 |alfa de FW. Acesta  |
   3 |este versiune de    |
   4 |dezvoltare.         |
   5 |Folosindaceasta     |
   6 |versiune nu este    |
   7 |recomandata si poate|
   8 |cauza deteriorarea  |
   9 |imprimantei         |

[E]: Text is longer than definition on line 2451: cols=20 rows=8 (rows diff=1)
 source text:
      ₀₁₂₃₄₅₆₇₈₉₀₁₂₃₄₅₆₇₈₉
   1 |You are using       |
   2 |firmware beta       |
   3 |version. This is    |
   4 |development version.|
   5 |Using this version  |
   6 |is not recommended  |
   7 |and may cause       |
   8 |printer damage.     |
 translated text:
      ₀₁₂₃₄₅₆₇₈₉₀₁₂₃₄₅₆₇₈₉
   1 |Utilizati versiunea |
   2 |beta de FW. Acesta  |
   3 |este versiune de    |
   4 |dezvoltare.         |
   5 |Folosindaceasta     |
   6 |versiune nu este    |
   7 |recomandata si poate|
   8 |cauza deteriorarea  |
   9 |imprimantei         |

```

**Comment by 3d-gussner** [comment]
If you can't shorten the messages `c=20 r=8` we can increase the rows slightly, but then think about how it looks at the last "page"
Example:
```
      ₀₁₂₃₄₅₆₇₈₉₀₁₂₃₄₅₆₇₈₉
   1 |Instrumentul de     |
   2 |filament solicitat  |
   3 |nu este disponibil  |
   4 |pe acest hardware.  |
      ₀₁₂₃₄₅₆₇₈₉₀₁₂₃₄₅₆₇₈₉
   5 |Verificati G-codul  |
   6 |pentru indicele     |
   7 |instrumentului in   |
   8 |afara intervalului  |
      ₀₁₂₃₄₅₆₇₈₉₀₁₂₃₄₅₆₇₈₉
   9 |(T0-T4).            |
```
This may look not that nice with the last page only showing one line `(T0-T4)`

**Comment by Hauzman** [comment]
@3d-gussner  I'm working on it

File: lang/po/Firmware_ro.po (hunk 1 above)
Comment (Hauzman) [reply lang/po/Firmware_ro.po]: Done. my bad :(

File: lang/po/Firmware_ro.po [hunk 2]
```diff
@@ -2119,26 +2136,29 @@ msgstr "Temperaturi"
...
 msgstr ""
+"Versiune FW MMU este incompatibil cu cea a imprimantei."
+"Asigurati-va ca FW MMU este actualizat."
 
 #. MSG_DESC_SELECTOR_CANNOT_HOME c=20 r=8
 #: ../../Firmware/mmu2/errors_list.h:202 ../../Firmware/mmu2/errors_list.h:242
 msgid ""
 "The Selector cannot home properly. Check for anything blocking its movement."
-msgstr ""
+msgstr "Selector-ul nu poate reveni" corect. Verificati daca exista ceva care blocheza"
```
Comment (3d-gussner) [inline lang/po/Firmware_ro.po]: Double quote issue
Here the fix
```
#. MSG_DESC_SELECTOR_CANNOT_HOME c=20 r=8
#: ../../Firmware/mmu2/errors_list.h:202 ../../Firmware/mmu2/errors_list.h:242
msgid ""
"The Selector cannot home properly. Check for anything blocking its movement."
msgstr ""
"Selector-ul nu poate reveni corect. Verificati daca exista ceva care blocheza"
"miscarea."
```

File: lang/po/Firmware_ro.po [hunk 3]
```diff
@@ -1730,6 +1745,8 @@ msgid ""
 "Requested filament tool is not available on this hardware. Check the G-code "
 "for tool index out of range (T0-T4)."
 msgstr ""
+"Instrumentul de Fil. solicitat nu este disponibil pe acest hardware. Verifica G-codul "
```
Comment (3d-gussner) [inline lang/po/Firmware_ro.po]: Would this be a valid translation? As this fits on tow pages with c=20 r=8.
```
"Instrumentul de Fil. solicitat nu este disponibil pe acest hardware. Verifica"
" G-codul pentru indexul sculei in afara intervalului (T0-T4)."
```

**Review by 3d-gussner — CHANGES_REQUESTED** [review]
Please fix the double quote issue.

Are you sure that some of the translations are the same as the Eglish one?

File: lang/po/Firmware_ro.po (hunk 2 above)
Comment (Hauzman) [reply lang/po/Firmware_ro.po]: Done

File: lang/po/Firmware_ro.po (hunk 3 above)
Comment (Hauzman) [reply lang/po/Firmware_ro.po]: @3d-gussner I have Change it to 
"Instrumentul de Fil. solicitat nu este disponibil pe acest hardware. Verifica"
"G-codul ptr indexul instrum. in afara intervalului (T0-T4)."

**Review by 3d-gussner — APPROVED** [review]

File: lang/po/Firmware_ro.po [hunk 4]
```diff
@@ -725,7 +727,7 @@ msgid ""
 "Filament sensor didn't switch off while unloading filament. Ensure filament "
 "can move and the sensor works."
 msgstr ""
-"Senzorul de filament nu sa oprint la scoaterea Fil. Asigurativa ca Fil."
+"Senzorul de filament nu sa oprint la descare Fil. Asigurativa ca Fil."
```
Comment (Ro3Deee) [inline lang/po/Firmware_ro.po:730]: the correct spelling: "Senzorul de filament nu s-a oprit la descarcarea fil. Asigurati-va ca fil."

File: lang/po/Firmware_ro.po [hunk 5]
```diff
@@ -743,8 +745,8 @@ msgid ""
 "Filament sensor triggered too early while loading to extruder. Check there "
 "isn't anything stuck in PTFE tube. Check that sensor reads properly."
 msgstr ""
-"FSensoz sa declansat prea devreme in timpul incarcarii in expruder. Verifica daca"
-"nu este nimic blocat in PTFE. Verifica daca senzorul functioneaza corect."
+"FSenzor sa declansat prea devreme in timpul incarc. in extrud. Verifica daca"
```
Comment (Ro3Deee) [inline lang/po/Firmware_ro.po:748]: "FSenzor s-a declansat prea devreme in timpul incarc. in extrud. Verifica daca"

File: lang/po/Firmware_ro.po [hunk 6]
```diff
@@ -408,7 +408,7 @@ msgstr "Taie filamentul"
 #: ../../Firmware/messages.cpp:129 ../../Firmware/ultralcd.cpp:4282
 #: ../../Firmware/ultralcd.cpp:4287 ../../Firmware/ultralcd.cpp:4292
 msgid "Cutter"
-msgstr ""
+msgstr "Cutter"
```
Comment (Ro3Deee) [inline lang/po/Firmware_ro.po:411]: msgstr "Taietor"

File: lang/po/Firmware_ro.po [hunk 7]
```diff
@@ -559,42 +559,45 @@ msgstr "Autoload fil."
...
 #: ../../Firmware/mmu2/errors_list.h:148 ../../Firmware/mmu2/errors_list.h:186
 msgid "FILAMENT ALREADY LOA"
-msgstr ""
+msgstr "FILAM. DEJA INCARCAT"
 
 #. MSG_TITLE_FINDA_DIDNT_TRIGGER c=20
 #: ../../Firmware/mmu2/errors_list.h:118 ../../Firmware/mmu2/errors_list.h:156
 msgid "FINDA DIDNT TRIGGER"
-msgstr ""
+msgstr "FINDA NU SA DECLANST"
```
Comment (Ro3Deee) [inline lang/po/Firmware_ro.po:578]: msgstr "FINDA NU S-A DECLANSAT"

**Pull request merged**